packages/auth-js/README.md

```diff
@@ -0,0 +1,142 @@
+# Auth js middleware for Hono
```

Comment:
`Auth js` should be `Auth.js`.

Please fix the other parts as well.

Reply:
@yusukebe  should be good now